frontend/webapp/containers/main/overview/overview-data-flow/build-destination-nodes.ts

```diff
@@ -73,7 +73,7 @@ export const buildDestinationNodes = ({ loading, entities, positions, unfiltered
         nodeWidth,
         type: OVERVIEW_NODE_TYPES.ADD_DESTIONATION,
```

Comment:
```suggestion
        type: OVERVIEW_NODE_TYPES.ADD_DESTINATION,
```

is this a variable/const that's defined somewhere? I noticed that too but wasn't sure how to update it

Reply:
Yes it's a const, I fixed the typo thanks :)